tests/test_http.py

```diff
@@ -0,0 +1,22 @@
+"""Tests for the safir.http module.
```

Comment:
Nit: `pytest` allows either `test_foo.py` or `foo_test.py` naming, but the latter is much friendlier to tab completion, so I usually standardize on the latter for my projects. The semantic hint for humans that these are test files is usually adequately provided by being in the `tests` directory, so not having the `test_` part first isn't missed (at least in my experience).

Reply:
Good idea. I didn't know that it was an option, but I've switched it now.